### PR TITLE
Added vpc_access parameter to RevisionTemplate object. Added test for 'chaosgcp.cloudrun.actions.update_service'

### PR DESCRIPTION
**Pull request by elenamares**

### CODE REVIEW

**Comment by elenamares** [comment]
Sylvain, I give up! I don't understand what lint wants.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,15 @@
 ## [Unreleased][]
 
 [Unreleased]: https://github.com/chaostoolkit-incubator/chaostoolkit-google-cloud-platform/compare/0.5.0...HEAD
+### Changed - 2022-12-30
```
Comment (Lawouach) [inline CHANGELOG.md]: no need for the date here

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,15 @@
...
+- Renamed field max_instance_request_concurrency in create_service method. Modified init value.
+### Changed - 2022-12-19
```
Comment (Lawouach) [inline CHANGELOG.md]: no need to set the `### Changed` header as it's already set above

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,15 @@
...
+- Added vpc_access parameter to RevisionTemplate object
+- Added test for 'chaosgcp.cloudrun.actions.update_service'
+
+### Changed
```
Comment (Lawouach) [inline CHANGELOG.md]: no need to set the `### Changed` header as it's already set above

**Review by Lawouach — CHANGES_REQUESTED** [review]
A few minor comments.

**Review by elenamares — COMMENTED** [review]
Fixed CHANGELOG.md as per comments.

File: CHANGELOG.md (hunk 2 above)
Comment (elenamares) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 1 above)
Comment (elenamares) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 3 above)
Comment (elenamares) [reply CHANGELOG.md]: Fixed.

**Review by Lawouach — APPROVED** [review]
LGTM

**Pull request merged**